apps/api/modules/answers/answers.routes.ts

```diff
@@ -81,7 +83,7 @@ const answersPlugin: FastifyPluginAsync = async (fastify) => {
 
 			try {
 				const answer = await fastify.db.questionAnswer.create({
-					data: { content, questionId: id, createdById: sessionData._user.id },
+					data: { questionId: id, createdById: sessionData._user.id, ...body },
```

Comment:
Tak samo tutaj, po co ten spread? To tylko rodzi problemy – przekazujemy do prismy coś, ale nie wiemy dokładnie co to jest…